File: Python/exercicio-17/solucao.py
def main():
  def multa_ou_nao():
    velocidade_carro = int(input('Digite a velocidade do carro: ' ))
    velocidade_excedida = velocidade_carro - 80
    valor_multa = velocidade_excedida * 5
    if velocidade_carro > 80:
      print(f'Você excedeu {velocidade_excedida}kms e vai pagar {valor_multa} reais de multa.')
    else:
      print("Parabéns voce não foi multado !")

  multa_ou_nao()

      
main()

  

File: JavaScript/exercicio-17/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {

  function multaOuNao() {

    const velocidadeCarro = parseInt(prompt("Digite a velocidade do carro: " ));
    const velocidadeExcedida = velocidadeCarro - 80
    const valorMulta = velocidadeExcedida * 5

    if (velocidadeCarro > 80) {
      console.log(`Você excedeu ${velocidadeExcedida}kms e vai pagar ${valorMulta} reais de multa.`)
    } else {
      console.log("Parabéns voce não foi multado !")
    }
  }

  multaOuNao();

}

main();
readline.question('Pressione Enter para sair ... ')